kola/tests/fleet/fleet.go

```diff
@@ -136,19 +137,21 @@ func Proxy(c platform.TestCluster) error {
 	var status []byte
 
 	// 5 seconds seems like a reasonable amount of time to make this happen.
```

Comment:
I think this comment is no longer relevant here. Deleting or moving down to the util.Retry call is probably fine.